service.languagepreferencemanager/.gitignore

```diff
@@ -0,0 +1,39 @@
+*.py[cod]
```

Comment:
please drop from submission

Reply:
Thanks for the review!
Should I drop the whole .gitignore file or just the 1st line ?

I'll do that before resubmitting the PR to Nexuw according to you other message.

Reply:
whole file